kano_updater/ui/install_window.py

```diff
@@ -31,7 +32,11 @@ def __init__(self):
         apply_styling_to_screen(self.CSS_FILE)
 
         Gtk.Window.__init__(self)
-        self.fullscreen()
+        # self.fullscreen()
+        screen = Gdk.Screen.get_default()
+        width = screen.get_width()
+        height = screen.get_height()
+        self.set_size_request(width, height + 80)
```

Comment:
Is this for the Gtk fullscreen hack? If so, can we have a comment here to explain and does the height really need to be `+ 80` can't you just add `1`?